pkg/apis/security/v1alpha1/types.go

```diff
@@ -77,6 +77,9 @@ type Rule struct {
 	// destinations.
 	// +optional
 	To []NetworkPolicyPeer `json:"to"`
+	// RuleName describes the intention of this rule.
+	// +optional
+	RuleName string `json:"ruleName"`
```

Comment:
did not update here either?

Reply:
I'm going to push them with name unique validation together.
Just about to do that lol.